lib/Service/RoomService.php

```diff
@@ -524,4 +542,43 @@ public function verifyPassword(Room $room, string $password): array {
 			'url' => '',
 		];
 	}
+
+	public function setExpireInterval(Room $room, Participant $participant, int $seconds): void {
+		$event = new ChangeExpireDateEvent($room, $seconds);
+		$this->dispatcher->dispatch(Room::EVENT_BEFORE_SET_EXPIRE_DATE, $event);
```

Comment:
Event name sounds like the room would be expired instead of the chat messages.
(And room expiration might come in the future, so we should keep the event name clear)